src/Compilers/CSharp/Portable/Syntax/EventFieldDeclarationSyntax.cs

```diff
@@ -1,11 +0,0 @@
-﻿
-namespace Microsoft.CodeAnalysis.CSharp.Syntax
-{
-    public sealed partial class EventFieldDeclarationSyntax : BaseFieldDeclarationSyntax
-    {
-        public EventFieldDeclarationSyntax AddDeclarationVariables(params VariableDeclaratorSyntax[] items)
```

Comment:
They are now generated by the syntax generator, as they used to be.

---
In reply to: [73446195](https://github.com/dotnet/roslyn/pull/12917#discussion_r73446195) [](ancestors = 73446195)